apstra/two_stage_l3_clos_configlets.go

```diff
@@ -0,0 +1,174 @@
+package apstra
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+const (
+	apiUrlBlueprintConfiglets       = apiUrlBlueprintById + apiUrlPathDelim + "configlets"
+	apiUrlBlueprintConfigletsPrefix = apiUrlBlueprintConfiglets + apiUrlPathDelim
+	apiUrlBlueprintConfigletsById   = apiUrlBlueprintConfigletsPrefix + "%s"
+)
+
+type rawTwoStageL3ClosConfiglet struct {
+	Data      rawConfigletData `json:"configlet"`
+	Id        string           `json:"id"`
+	Condition string           `json:"condition"`
+	Label     string           `json:"label"`
+}
+
+type TwoStageL3ClosConfiglet struct {
+	Data      ConfigletData
+	Id        string
```

Comment:
`Id` here and in `rawTwoStageL3ClosConfiglet` should be type `ObjectId`.

This will eliminate some type casting elsewhere too.